functions/nodejs-apps.bash

```diff
@@ -105,40 +105,57 @@ frontail_setup() {
   if ! cond_redirect chmod 644 /etc/systemd/system/frontail.service; then echo "FAILED (permissions)"; return 1; fi
   if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
   if ! cond_redirect systemctl enable --now frontail.service; then echo "FAILED (enable service)"; return 1; fi
-  if cond_redirect systemctl restart frontail.service; then echo "OK"; else echo "FAILED (restart service)"; return 1; fi # restart the service to make the change visible
+  if cond_redirect systemctl restart frontail.service; then echo "OK"; else echo "FAILED (restart service)"; return 1; fi # Restart the service to make the change visible
 
   if openhab_is_installed; then
     dashboard_add_tile "frontail"
   fi
 }
 
-## Function for adding a user specifed log to frontail
+## Function for adding/removing a user specifed log to/from frontail
 ##
-##    add_frontail_log()
+##    custom_frontail_log()
 ##
-add_frontail_log() {
-  if [[ -z $INTERACTIVE ]]; then
-    return 0
-  fi
-
+custom_frontail_log() {
   local frontailService="/etc/systemd/system/frontail.service"
-  local newLog
+  local addLog
+  local removeLog
+  local array
 
-  if [[ -f $frontailService ]]; then
-    if ! newLog="$(whiptail --title "Enter file path" --inputbox "\\nEnter the path to the logfile that you would like to add to frontail:" 9 80 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
+  if ! [[ -f $frontailService ]]; then
+    whiptail --title "Frontail not installed" --msgbox "Frontail is not installed!\\n\\nCanceling operation!" 9 80
+    return 0
+  fi
 
-    if [[ -f $newLog ]]; then
-      echo -n "$(timestamp) [openHABian] Adding '${newLog}' to frontail... "
-      if ! cond_redirect sed -i -e "/^ExecStart/s/$/ ${newLog}/" $frontailService; then echo "FAILED (add log)"; return 1; fi
-      if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
-      if cond_redirect systemctl restart frontail.service; then echo "OK"; else echo "FAILED (restart service)"; return 1; fi
+  if [[ $1 == "add" ]]; then
```

Comment:
Does it work unattended ? Haven't seen it being called. What's the openhabian.conf parameter to set ?

Note that if not INTERACTIVE there will not be a $1 and if so this test is superfluous/unneeded

Reply:
Nope, apparently I forgot to do that part and only changed the function itself, I'll fix that.